server/repository/migrations/postgres/2022-06-21-013225_create_log_table/up.sql

```diff
@@ -16,9 +17,10 @@ CREATE TYPE activity_log_type AS ENUM (
     'REQUISITION_STATUS_FINALISED'
 );
 
-CREATE TABLE activity_log (
+CREATE TABLE activity_log
+(
     id TEXT NOT NULL PRIMARY KEY,
-    type log_type, 
+    type activity_log_type,
```

Comment:
offroad - was unable to create a new pg db due to this being incorrectly named

Reply:
🙏